### PR TITLE
Update modes-of-operation.rst

### PR DESCRIPTION
**Pull request by paulhoogsteder**
Clarified usage of SOA record, NS records and supermasters table.

### Short description
Clarified usage of SOA record, NS records and supermasters table.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [n/a ] compiled this code
- [n/a ] tested this code
- [X] included documentation (including possible behaviour changes)
- [n/a] documented the code
- [n/a] added or modified regression test(s)
- [n/a] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/modes-of-operation.rst [hunk 1]
```diff
@@ -206,9 +206,9 @@ must be met:
 
 
 - :ref:`setting-superslave` support must be enabled
-- The supermaster must carry a SOA record for the notified domain
-- The supermaster IP must be present in the 'supermaster' table
-- The set of NS records for the domain, as retrieved by the slave from the supermaster, must include the name that goes with the IP address in the supermaster table
+- The supermaster must carry a SOA record for the notified domain corresponding with one of the NS records for that domain
```
Comment (Habbie) [inline docs/modes-of-operation.rst]: This sentence was correct before you changed it - the name in the SOA is ignored.

File: docs/modes-of-operation.rst [hunk 2]
```diff
@@ -206,9 +206,9 @@ must be met:
...
+- The supermaster IP must be present in the 'supermasters' table in the database on the slave, along with the name mentioned in the SOA
```
Comment (Habbie) [inline docs/modes-of-operation.rst]: no, along with any name that is in the NS set

File: docs/modes-of-operation.rst [hunk 3]
```diff
@@ -206,9 +206,9 @@ must be met:
...
+- The set of NS records for the domain, as retrieved by the slave from the supermaster, must include the name that goes with the IP address in the supermasters table
```
Comment (Habbie) [inline docs/modes-of-operation.rst]: Yes!

**Comment by pieterlexis** [comment]
I took the liberty to update the PR fixing the comments from @Habbie

**Pull request merged**